test/e2e/service.go

```diff
@@ -567,10 +577,12 @@ var _ = Describe("Services", func() {
 		jig.TestReachableUDP(nodeIP, udpNodePort, kubeProxyLagTimeout)
 
 		By("hitting the TCP service's LoadBalancer")
-		jig.TestReachableHTTP(tcpIngressIP, svcPort, loadBalancerLagTimeout)
+		jig.TestReachableHTTP(tcpIngressIP, svcPort, loadBalancerCreateTimeout)
```

Comment:
this should be LagTimeout - we've already proven that the API says there is a LB.